notebooks/idr0094-r.ipynb

```diff
@@ -0,0 +1,1441 @@
+       "9788                                                             ESP0025976\n",
+       "9789                                                             ESP0025976\n",
+       "9790                                                             ESP0025976\n",
+       "9791                                                             ESP0025976\n",
+       "9792                                                             ESP0025976\n",
+       "     Well Name\n",
+       "1    a1       \n",
+       "2    a2       \n",
+       "3    a3       \n",
+       "4    a4       \n",
+       "5    a5       \n",
+       "6    a6       \n",
+       "7    a7       \n",
+       "8    a8       \n",
+       "9    a9       \n",
+       "10   a10      \n",
+       "11   a11      \n",
+       "12   a12      \n",
+       "13   b1       \n",
+       "14   b2       \n",
+       "15   b3       \n",
+       "16   b4       \n",
+       "17   b5       \n",
+       "18   b6       \n",
+       "19   b7       \n",
+       "20   b8       \n",
+       "21   b9       \n",
+       "22   b10      \n",
+       "23   b11      \n",
+       "24   b12      \n",
+       "25   c1       \n",
+       "26   c2       \n",
+       "27   c3       \n",
+       "28   c4       \n",
+       "29   c5       \n",
+       "30   c6       \n",
+       "⋮    ⋮        \n",
+       "9763 f7       \n",
+       "9764 f8       \n",
+       "9765 f9       \n",
+       "9766 f10      \n",
+       "9767 f11      \n",
+       "9768 f12      \n",
+       "9769 g1       \n",
+       "9770 g2       \n",
+       "9771 g3       \n",
+       "9772 g4       \n",
+       "9773 g5       \n",
+       "9774 g6       \n",
+       "9775 g7       \n",
+       "9776 g8       \n",
+       "9777 g9       \n",
+       "9778 g10      \n",
+       "9779 g11      \n",
+       "9780 g12      \n",
+       "9781 h1       \n",
+       "9782 h2       \n",
+       "9783 h3       \n",
+       "9784 h4       \n",
+       "9785 h5       \n",
+       "9786 h6       \n",
+       "9787 h7       \n",
+       "9788 h8       \n",
+       "9789 h9       \n",
+       "9790 h10      \n",
+       "9791 h11      \n",
+       "9792 h12      "
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "screen <- loadObject(server, \"ScreenData\", screenId)\n",
+    "\n",
+    "annos <- getAnnotations(server, 'ScreenData', getOMEROID(screen), nameFilter = tableName)\n",
+    "annotationFileID = as.integer(annos$FileID)\n",
+    "\n",
+    "df <- loadDataframe(screen, annotationFileID)\n",
+    "df"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Rename the columns to simplify exploration "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "names(df)[24] <- \"ConcX\"\n",
+    "names(df)[25] <- \"RespY\"\n",
```

Comment:
Why is this column renamed to "RespY" ? According to what I see from the dataframe on the notebook, the column 25 is the one with the "Percentage inhibition DPC" value ? This means these numbers will end up on the Y axis ?
What is RespY an abbreviation of ? Maybe I do not fully understand the flow of data here.